include/Core/DB/TableManager.php

```diff
@@ -267,4 +267,29 @@ public function delete_logs_older_than( $interval_in_days ) {
 		return $deleted_rows_count;
 	}
 
+	/**
+	 * Fetch logs items to export as CSV.
+	 *
+	 * @param array $ids Optional. List of Log IDs to be fetched. Fetches all logs when param is optional.
+	 * @return array     Log items.
+	 */
+	public function fetch_log_items_to_export( $ids = array() ) {
```

Comment:
This function as such looks fine, but I am trying to think whether we can refactor the existing `fetch_log_items` function and reuse it instead of creating an almost similar function..